### ISSUE TITLE
AskTim Problem/Video Screenreader Improvements

### ISSUE THREAD

**Issue by ChristopherChudzicki** [mitodl/smoot-design#229]
### Description/Context
Feedback from Rich based on testing with NVDA

1. **Flashcards:** Currently, the next/previous buttons focus the card flashcard automatically, but the card is sometimes (always?) refocused before the new content has been rendered, so new card isn't read aloud.
    - Suggested Fix: Do not automatically focus the flashcard.
    - Make the card button itself a live region with `aria-live="polite"`
2. **Tabs, Issue 1:** Selecting a tab has inconsistent behavior. Sometimes focus remains on the tab (Chat, Summary) but sometimes it moves to the tab panel.  The focus should consistently stay on the tab. Moving focus is surprising.
3. **Tabs, Issue 2** Moving the arrow keys on our tabs does not focus the tab, i.e., "selection does not follow focus". WCAG says selection following focus is optional, but is [recommended](https://www.w3.org/WAI/ARIA/apg/practices/keyboard-interface/#kbd_selection_follows_focus) when tabpanels content is already loaded.
    - Suggested fix: Make selection follow focus
    - Note: The current behavior is MUI's default, presumably because the tabpanel might load slowly, in which case the behavior is recommended against, I guess. 

### Plan/Design
The above changes seem straightforward and often simpler than what we do (e.g., not moving focus).

Our MUI tabs have a `selectionFollowsFocus` prop.

**Comment by ChristopherChudzicki** [comment]
An additional piece of feedback from Rich:
> when I ask a question, I get two copies of the response: one before the input box, and one after. Essentially the response is repeated in a separate container marked as a live region. why not mark the normal output area as a live region which will eliminate the need for two copies of the response!

The chat component includes messages followed by an input field followed by a dedicated aria-live region for announcing new messages. So...why have a separate aria-live region? Why not just make the main message area aria-live? To be honest, it's a pattern we followed from other sites. I **think** we tried having the main message area be aria-live, and my recollection is that it didn't work well with streamed content:
- the LLM messages are streamed, individual words (or small numbers of words) being sent to the browser continuously 
- The current aria-live region only reads the text once it has finished streaming

We may want to verify that including `aria-live` on the main message area doesn't work well.

For comparison with other LLM sites: claude.ai has no aria-live, chatgpt.com does it the same way as us. (They actually have two aria-live regions, unsure what the second is for. Maybe generic errors / announcements?). Curious what others do.

**One issue with our current implementation:** is that the aria-live region actually contains the raw markdown. So it will read "star star bold text star star". 

**TODO?** As part of the above fixes, it would be nice to (A) verify that my recollection above is correct. Maybe it isn't. (B) document this choice with some comment in the code, and (C) fix the markdown reading issue.
